packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -251,14 +255,10 @@ export const InputBaseInput = styled('input', {
   };
 });
 
-const inputGlobalStyles = (
-  <GlobalStyles
-    styles={{
-      '@keyframes mui-auto-fill': { from: { display: 'block' } },
-      '@keyframes mui-auto-fill-cancel': { from: { display: 'block' } },
-    }}
-  />
-);
+const InputGlobalStyles = globalCss({
```

Comment:
We should include this change in the API in the migration guide, if we haven't done it already.

Reply:
Can you explain more? I don't see any migration for InputBase. If you mean `globalCss`, I plan to merge this one first and then open a new PR for Pigment CSS opt-in guide.

Reply:
Nevermind, this is used internally, I was thinking this will be the API we document for developers. As long as we have the same public API, it should be fine

Reply:
I just noticed that the same function is called `injectGlobal` in emotion. It should be fine though since we don't expose this through `@mui/system`